src/static/client/app/signup/signup.component.scss

```diff
@@ -0,0 +1,110 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+  
+  body, html {
+    height: 100%;
+    font-family: Arial, sans-serif;
+  }
+  
+  .signup {
+    background: linear-gradient(to right, #1632aa, #B30D2C);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+
+    .button-container {
+      display: flex;
+      justify-content: space-between;
+    }
+    
+    #icon {
+      width: 150px;  // Adjust as needed to change the size of the logo.
+      height: auto;
+      display: block;
+      margin: 0 auto 20px auto; // This centers the image and adds space below it.
+    }
+  
+    .wrapper {
+      max-width: 400px;
+      background-color: #ffffff;
+      border-radius: 10px;
+      box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
+      padding: 30px;
+      width: 100%;
+      max-width: 350px;
+  
+      h2 {
+        text-align: center;
+        margin-bottom: 24px;
+      }
```

Comment:
Its not good practice to edit all h2 elements. HTML elements shouldn't be tied to styles usually, but more so how the page is organized to help with accessibility :). You can change this to the bootstrap .h2 class (which you could adjust directly or perhaps use a sass function as seen in their documentation: https://getbootstrap.com/docs/5.0/customize/sass/)

Reply:
Yes you are right changed it